apps/pie-docs/src/assets/styles/components/_nav.scss

```diff
@@ -60,8 +68,9 @@ $nav-item-active-left-padding:        calc($nav-item-left-padding - $nav-item-ac
 .c-nav-category {
   display: flex;
   align-items: center;
-  gap: f.spacing(d);
-  padding: $nav-category-padding;
+  gap: $nav-category-gap;
+  padding: 10px f.spacing(d);
```

Comment:
if we're using non-standard whitespace measurements (i.e. 10) I would talk to @jordifornellsJET about whether or not this is correct